evap/evaluation/views.py

```diff
@@ -163,15 +163,18 @@ def legal_notice(request):
 def contact(request):
     message = request.POST.get("message")
     title = request.POST.get("title")
-    email = request.user.email or f"User {request.user.id}"
+    email = (
+        settings.DEFAULT_FROM_EMAIL
+        if request.POST.get("anonymous") == "on"
```

Comment:
Can't do this: Users would be able to send a POST request with `anonymous=on` even if anonymous feedback was disabled in the settings. We have to validate the input here.

Probably best to just throw a SuspiciousOperation if anonymous was requested, but is disabled in the settings.

Reply:
we could also test this :)